index.js

```diff
@@ -107,7 +107,7 @@ function handleArgs(command, args, options = {}) {
 }
 
 function handleInput(spawned, input) {
-	if (input === undefined) {
+	if (input === undefined || spawned.stdin === undefined) {
```

Comment:
Can you add a comment on it being a workaround for https://github.com/nodejs/node/issues/26852 ?

Reply:
done